benchmark/net/http_server_for_chunky_client.js

```diff
@@ -0,0 +1,45 @@
+"use strict";
+var path = require('path');
+var http = require('http');
+var fs = require('fs');
+var spawn = require('child_process').spawn;
+var common = require('../common.js')
+var test = require('../../test/common.js')
+var pep = path.dirname(process.argv[1])+'/chunky_http_client.js';
+var PIPE = test.PIPE;
+
+var server;
+
+fs.unlinkSync(PIPE);
+server = http.createServer(function(req, res) {
+  res.writeHead(200, { 'content-type': 'text/plain',
+                       'content-length': '2' });
+  res.end('ok');
+});
+
+server.on('error', function(err) {
+  console.log("Error:");
+  console.log(err);
+});
+
+try {
+  server.listen(PIPE, 'localhost');
+} catch(e) {
+  console.log("oops!");
+  console.log(e);
+}
```

Comment:
Happy to Trevor and BTW thanks for the guidance and additional comments they are quite helpful.  Even though we do an unlinkSync(PIPE) above, it seems to be no guarantee against another mistaken process interfering accidentally.  I seem to remember that it is possible to bind to a port with an AF_INET address exclusively or not a la SO_REUSEADDR / etc and I had assumed there was some similar risks with AF_UNIX family that another process might accidentally do.  Please let me know if you believe otherwise (e.g. call cannot ever fail) and I would like to find out more as well and simplify this code.